src/main/scala/cromwell/binding/Call.scala

```diff
@@ -31,13 +31,16 @@ case class Call(alias: Option[String], taskFqn: FullyQualifiedName, task: Task,
     else throw new UnsupportedOperationException("parent is write-once")
   }
 
+  private def unsatisfiedTaskInputs: Seq[TaskInput] = task.inputs.filterNot {case i => inputMappings.contains(i.name)}
   /**
    * Map of task-local input names to type for every input that this
```

Comment:
Yes I can see how that is confusing.  The comment is a little outdated.  Should probably read:

> Returns a Seq[WorkflowInput] representing the inputs to the call that are needed before its command can be constructed.  This excludes inputs that are satisfied via the 'input' section of the Call definition.